source/connection-strings/mongodb-database-connection-strings.txt

```diff
@@ -48,9 +46,66 @@ On-Premises Connection URI
 
    mongodb://username:password@host:port/database
 
-For example, to use an account named ``migrator-service`` to connect to the
-``MongoEnterprises`` database:
+For example, to use an account named ``migrator-service`` with password
+``hunter2`` to connect to the ``MongoEnterprises`` database:
 
 .. code-block::
 
-   mongodb://migrator-service:password@localhost:27017/MongoEnterprises
\ No newline at end of file
+   mongodb://migrator-service:hunter2@localhost:27017/MongoEnterprises
+
+.. _rm-x509-auth:
+
+Using X.509 Authentication
+---------------------------
+
+To connect to MongoDB using X.509 authentication, omit a username and password.
+Instead, specify the on-premises host and port or the Atlas cluster URL and set 
+the following options:
+
+.. code-block::
+
+   <on-premises host:port or clusterurl.mongodb.net>?authSource=$external&authMechanism=MONGODB-X509&tlsCertificateKeyFile=/path/to/cert/client.p12&tlsCertificateKeyFilePassword=certpassword
+
+
+.. list-table::
```

Comment:
Does this also need `tls=true`? It wasn't mentioned in the word doc

Reply:
I don't believe it's a pre-requisite (rather, it probably falls back to it if x.509 is on).